.github/workflows/ci.yml

```diff
@@ -39,11 +38,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
Since it’s a package that will run in environments other than Linux, it will also be good to run the tests on these other platforms.

Reply:
One thing bothers me here. If someone creates an issue or there is an issue with this config and it's strictly related to Windows or MacOS, do we plan to help/fix it as well?

Reply:
Yep, because there are many other people who don't use Linux as their development system, and since this package is a CLI, it wouldn't hurt to run tests to ensure it works well on those other systems.

Reply:
I'm ok if helping with user config is common in express org